test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] You did a great job 😃 at adding the test file at the gitignore, but I see that the file of the test is pushed to the GitHub this is because maybe you git push first after that you put the file in the git ignore and this is wrong 😃and this not from the requirements 💯, so kindly : 
- Try to 😊put the file test firstly in the gitignore after that you can push to not put the test file to the GitHub